src/App.tsx

```diff
@@ -1,17 +1,19 @@
 import apolloLogo from "./assets/apollo-logo.jpeg";
-
-import "./App.css";
+import styles from "./App.module.css";
+import classnames from "classnames";
 
 function App() {
   return (
-    <>
-      <div>
-        <a href="https://apollo.io" target="_blank">
-          <img src={apolloLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
+    <div className={styles.root}>
+      <a href="https://www.apollo.io/tech-blog" target="_blank">
+        <img
+          src={apolloLogo}
+          className={classnames(styles.logo)}
+          alt="Apollo logo"
+        />
+      </a>
       <h1>Thanks for interviewing with Apollo!</h1>
```

Comment:
What about a simple indicator for them to place their code like:

```suggestion
      <h1>Thanks for interviewing with Apollo!</h1>
      &#8249;Your code here&#8250;
```
Which renders as `<Your code here>`.

Reply:
Done!